From 45e63dd9ae1df8fb900658cda5bce90da202d479 Mon Sep 17 00:00:00 2001
From: Sai Hari Chandana Kalluri <chandana.kalluri@xilinx.com>
Date: Fri, 8 Nov 2019 14:37:50 -0800
Subject: [PATCH] build: always disable platform guessing.

Upstream-status: Inappropriate

Signed-off-by: Ismo Puustinen <ismo.puustinen@intel.com>

NOTE: the original patch for thud release is taken from
meta-maker recipe. However for zeus release update/rebase the patch as
setup.py for python-pillow has changed

Signed-off-by: Sai Hari Chandana Kalluri <chandana.kalluri@xilinx.com>
---
 setup.py | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/setup.py b/setup.py
index 76bdfb1..a1e24ce 100755
--- a/setup.py
+++ b/setup.py
@@ -425,7 +425,7 @@ class pil_build_ext(build_ext):
         #
         # add platform directories
 
-        if self.disable_platform_guessing:
+        if self.disable_platform_guessing or True:
             pass
 
         elif sys.platform == "cygwin":
@@ -505,16 +505,6 @@ class pil_build_ext(build_ext):
 
         # FIXME: check /opt/stuff directories here?
 
-        # standard locations
-        if not self.disable_platform_guessing:
-            _add_directory(library_dirs, "/usr/local/lib")
-            _add_directory(include_dirs, "/usr/local/include")
-
-            _add_directory(library_dirs, "/usr/lib")
-            _add_directory(include_dirs, "/usr/include")
-            # alpine, at least
-            _add_directory(library_dirs, "/lib")
-
         if sys.platform == "win32":
             if PLATFORM_MINGW:
                 _add_directory(
-- 
2.7.4

